unit-tests/syncer/test-ts-desync-2.py

```diff
@@ -0,0 +1,168 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+import pyrealsense2 as rs
+from rspy import log, test
+import sw
+
+
+#
+# This is a reproduction of a problem we saw with externally-synced cameras, where the master
+# camera is OK but the slave cameras generate delayed color frames (Depth at time X arrives 4ms
+# sooner than Color at time X-18ms) with exaggerated next-expected because of low-calculated-fps
+# (29 versus the nominal 30 that's supposed to be seen).
+#
+# This should no longer happen as recorded because the captured FPS, and therefore estimated
+# next-expected, is much more accurate: part of the problem was that ACTUAL_FPS was crudely
+# misrepresented (29.98 became 29). Nonetheless, with the below capture times, this is the behavior
+# we expect.
+#
+# See [RSDSO-19336]
+#
+
+
+sw.fps_d = 30
+sw.fps_c = 30
+sw.init()
+sw.start()
+
+#############################################################################################
+with test.closure( "Init" ):
+    # It can take a few frames for the syncer to actually produce a matched frameset (it doesn't
+    # know what to match to in the beginning)
+
+    # (sync.cpp:396) ... missing Color/66, next expected 63231.348758
+    sw.generate_color_frame( frame_number=1730, timestamp=63198.01542, next_expected=63231.348758 )
```

Comment:
This is some kind of regression no?
Before the changes the syncer could output this first set?